usage/src/main/java/com/cloud/usage/UsageManagerImpl.java

```diff
@@ -331,7 +331,9 @@ public boolean start() {
             _sanity = _sanityExecutor.scheduleAtFixedRate(new SanityCheck(), 1, _sanityCheckInterval, TimeUnit.DAYS);
         }
 
+        _usageJobDao.removeLastOpenJobsOwned(_hostname, 0);
```

Comment:
@sureshanaparti , I think this should be done before the scheduling to prevent a race condition. It is unlikely but not impossible that we delete a job here that we just scheduled.
Do you agree?

Reply:
updated @DaanHoogland